pepper-apis/src/main/java/org/broadinstitute/ddp/export/DataExportCoordinator.java

```diff
@@ -31,13 +39,17 @@
 
     public static final int DEFAULT_BATCH_SIZE = 100;
     public static final int READER_BUFFER_SIZE_IN_BYTES = 10 * 1024;
+    public static final String RGP_GUID = "RGP";
+    public static final String RGP_ACTIVITY = "ENROLLMENT";
```

Comment:
To me, this is a red flag. Seeing study-specific IDs hardcoded in source code doesn't seem right to me. We're building special code for a specific study.

Reply:
We do have code that mentions some studies, but those are mostly utility tools / command-line tools that we developers run, or some unit tests. In other words, code that are not actual production code.

Reply:
I think the above mostly addresses this, but I'm going to at least move this stuff to the config file.  It's still ugly, though.

Reply:
I'll clean this piece up and then add Zim as a reviewer to make sure that this is the approach he suggested.